frontend/src/components/project/ProjectPageRoot.js

```diff
@@ -306,12 +306,12 @@ export default function ProjectPageRoot({
     if (!initiallyCaughtFollowers) {
       updateFollowers();
```

Comment:
```suggestion
      await updateFollowers();
```

The problem here seems to be that you don't wait for updateFollowers to finish. 
I think it makes more sense to leave it here and use the await keyword. Same below with likes